.gitignore

```diff
@@ -46,6 +46,7 @@ src/*/parser/parsetab.py
 
 # Venv
 .env
+venv
```

Comment:
Just curious, why did you add this? How does this directory get created?

Reply:
This is where I put the venv i create manually, I might have to to keep that change on my machine, it doesn't concern anyone else I suppose

Reply:
(Yes I still create virtualenv manually, for some reason virtualenvwrapper wouldn't create me environment with the right python version, and given that I don't find it too painful to make the commands manually, I continue that way)

Reply:
All right. I'm not against merging this in. A directory named `venv` in the root of the project is a sufficiently common scenario to allow it in the gitignore in my opinion.
Have you tried using the `-p` parameter to specify a python version? `mkvirtualenv -p python3.6 myenv`.

Reply:
Revert in https://github.com/inmanta/inmanta/pull/2475/commits/ea3ff86634f5f1e0c5a133527a3a9a5da05c6f3f

Reply:
I already tried that yes, but for some reason the python version was still 3.8, I didn't search really long what was going wrong to be honest, and now with the advice of bart I will not need those venv anymore